fix(wakatime-docs): broken `properties` table + add subsection for URL with Env. Var. (#3513)
JanDeDobbeleer/oh-my-posh: website/docs/segments/wakatime.mdx

| --------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `url`           | `string`  | The Wakatime [summaries][wk-summaries] URL, including the API key. Example above. Defaults to ``.                                                                                                                 |
| `http_timeout`  | `int`     | The time (_in milliseconds_, `ms`) it takes to consider an http request as **timed-out**. Defaults to `20`. If no segment is shown, try increasing this timeout.                                                  |
| `cache_timeout` | `int`     | The time (_in minutes_, `m`) it takes to invalidate/renew the Wakatime segment output. Defaults to `10`. Setting this to `0` disables the cache and requests to the API everytime the user presses the Enter key. |

### Dynamic API Key

> If the developer or the user of the theme wants to use the reference of their Wakatime API key (_stored from their Environment Variables_) without statically declaring them, then the following modification can be done.

Reviewer: ```suggestion
If you don't want to include the API key into your configuration, the following modification can be done.
```

```json
"properties": {
  // highlight-next-line
  "url": "https://wakatime.com/api/v1/users/current/summaries?start=today&end=today&api_key={{ .Env.WAKATIME_API_KEY }}",
  "cache_timeout": 10,
  "http_timeout": 500
}
```

:::note 

> `WAKATIME_API_KEY` **is not a strict name to use** as an Environment Variable for this segment in particular, **any name is possible and acceptable** as long as it exists and matches from your Environment Variables and from the theme itself.

Reviewer: ```suggestion
`WAKATIME_API_KEY` is an example, **any name is possible and acceptable** as long as the environment variable exists and contains the API key value.
```
